tf2onnx/tfonnx.py

```diff
@@ -393,19 +391,30 @@ def conv_convert_inputs(ctx, node, with_kernel=False, new_kernel_shape=None,
     # kernel must to be transposed
     if with_kernel:
         parent = node.inputs[1]
-        # note: kernel may be used by multiple nodes,
-        # so even kernel is a const, transposing kernel can't be done statically.
-        # so "transpose" op is inserted here and will consider to remove it in later optimization phase if possible.
-        input_name = node.input[1]
-        transpose = ctx.insert_new_node_on_input(node, "Transpose", input_name)
-        transpose.set_attr("perm", HWCN_TO_NCHW)
-        transpose.inserted_nchw = True
-        transpose.skip_conversion = True
-        ctx.copy_shape(input_name, transpose.output[0])
-        new_shape = spatial_map(ctx.get_shape(input_name), HWCN_TO_NCHW)
-        ctx.set_shape(transpose.output[0], new_shape)
-        nodes.append(transpose)
-        parent.data_format = "NCHW"
+
+        need_transpose = True
```

Comment:
why not use if- else, instead of need_transpose?

Reply:
ah, because there are 2 conditions ... if cons and if consumers==1

Reply:
sorry...misunderstood